frontend/app/module/election/page/ElectionHomePage.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "react-router-dom";
+
+import { MockTest } from "app/component/MockTest";
+import { NavBar } from "app/component/navbar/NavBar";
+
+import { useElection } from "@kiesraad/api";
+
+export function ElectionHomePage() {
+  const { election } = useElection();
+
+  return (
+    <>
```

Comment:
Wondering if this page should have a `PageTitle`..

Reply:
Added in 623a7a1ad6c1fea13ddaba577f23e2f61d29b1a4